fault.go

```diff
@@ -21,10 +21,10 @@ func Wrap(err error, w ...Wrapper) error {
 		return nil
 	}
 
-	// the passed err might already have a location if it's a fault.New error, or it might not if it's another type of
+	// The passed err might already have a location if it's a fault. New error, or it might not if it's another type of
```

Comment:
this isn't a sentence boundary period, it's a Go source code snippet, as in `fault.New("some error")`

Reply:
quotes would make it more clear tbh

Reply:
Oh, I see